clients/cobol-lsp-vscode-extension/src/services/copybook/downloader/CopybookDownloaderForUss.ts

```diff
@@ -159,4 +141,52 @@ export class CopybookDownloaderForUss extends ZoweExplorerDownloader {
 
     return true;
   }
+
+  public async hasMember(
+    profileName: string,
+    uss: string,
+    copybookName: string,
+    extensions: string[] = [""],
+  ): Promise<boolean> {
+    const id = this.createId(profileName, uss);
+    if (this.memberListCache.has(id)) {
+      let member: string | undefined;
+      for (const ext of extensions) {
+        member = this.memberListCache
+          .get(id)
+          ?.find(
+            (member) =>
+              member.toUpperCase() ===
+              `${copybookName.concat(ext).toUpperCase()}`,
+          );
+      }
+      return member ? true : false;
+    }
```

Comment:
Make a function out of this block.
Also there should be a check in the loop, this way, you are just checking the last extension.